package.json

```diff
@@ -92,5 +92,14 @@
     "css",
     "live reload",
     "sync"
-  ]
+  ],
+  "bugs": {
+    "url": "https://github.com/browsersync/browser-sync/issues"
+  },
+  "main": "index.js",
+  "directories": {
+    "example": "examples",
+    "test": "test"
+  },
+  "license": "Apache-2.0"
```

Comment:
Array way of declaring it is deprecated.

https://github.com/npm/npm/pull/8179

I did not notice the array though, I'll fix it later today